src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public class IdentityLogger
+    {
+        public ILogger Logger { get; set; }
```

Comment:
I had this and reverted it since i felt the it should be a public set property but I agree. I'll make the change